### ISSUE TITLE
next release of scala-logging

### ISSUE THREAD

**Issue by sullis** [lightbend-labs/scala-logging#453]
I am using Scala-Logging 3.9.5 in a production application:

```
com.typesafe.scala-logging:scala-logging_2.12:3.9.5
```

The latest version (3.9.5) was released in May 2022.  

Are there plans to make another release soon? cc: @SethTisue

**Comment by SethTisue** [comment]
@SethTisue I don't plan to do a release myself (my involvement in this repo is minimal), but I'm available to help if the other maintainers get stuck

**Comment by SethTisue** [comment]
I'm also available to help anoint new volunteer maintainers.

**Comment by jxnu-liguobin** [comment]
In fact, I am not sure if a version should be released for it (when there are no feature updates). If necessary, can it be released once or twice a year?

**Comment by SethTisue** [comment]
with sbt-ci-release, releasing is so easy that it's easy to default to just doing it

@sullis is there a particular change or changes you were interested in seeing ship...?

**Comment by limansky** [comment]
Hi,

  Is the release with slf4j2 support planned?

**Comment by jxnu-liguobin** [comment]
Is it only necessary to have do a release?

**Comment by limansky** [comment]
Yes for me, but I'm not sure about other users. Maybe there was something important merged as well since the last release.

**Comment by kacpwoja** [comment]
Hi guys,

We are using Scala-Logging 3.9.5 for Scala 2.13 in production.
It's the newest version, but it has a compile dependency on Scala 2.13.8, which is vulnerable to [CVE-2022-36944](https://nvd.nist.gov/vuln/detail/cve-2022-36944).
I see in build.sbt that a newer version of Scala (2.13.16 to be exact) is already merged to main, just not released.

I think it would make sense to release a new version to get rid of the vulnerability?

**Comment by jxnu-liguobin** [comment]
Hi @SethTisue , CI still has issues.

**Comment by SethTisue** [comment]
@lrytz I no longer have the necessary admin-level access to update repo secrets; maybe you could try refreshing the publishing secrets?

**Comment by SethTisue** [comment]
I don't know. for the username+password, did you use the token and not the actual username+password? it's now required to always use the token

**Comment by lrytz** [comment]
Yeah, my personal sonatype central account has access to the `com.typesafe` namespace, which I assume includes the `com.typesafe.scala-logging` child namespace used here. I created new tokens and added those.

So I'll try with a new key.

**Comment by lrytz** [comment]
tried again, that seems to have worked. https://github.com/lightbend-labs/scala-logging/actions/runs/17737317684/job/50403414716

@jxnu-liguobin can you check if all looks fine? https://repo1.maven.org/maven2/com/typesafe/scala-logging/scala-logging_2.13/3.9.6/

**Comment by jxnu-liguobin** [comment]
> tried again, that seems to have worked. https://github.com/lightbend-labs/scala-logging/actions/runs/17737317684/job/50403414716再试一次，似乎奏效了。https://github.com/lightbend-labs/scala-logging/actions/runs/17737317684/job/50403414716
> 
> [@jxnu-liguobin](https://github.com/jxnu-liguobin) can you check if all looks fine? https://repo1.maven.org/maven2/com/typesafe/scala-logging/scala-logging_2.13/3.9.6/你能检查一下一切看起来是否正常吗？https://repo1.maven.org/maven2/com/typesafe/scala-logging/scala-logging_2.13/3.9.6/

Thank you, I will test if the new jar is working later today
